microprofile/tests/tck/tck-lra/src/test/resources/logging.properties

```diff
@@ -0,0 +1,3 @@
+handlers=java.util.logging.FileHandler, java.util.logging.ConsoleHandler
+# Global default logging level. Can be overriden by specific handlers and loggers
+.level=DEBUG
```

Comment:
DEBUG is not a valid log level for Java util logging. Also using DEBUG or `FINEST` is not a good idea in tests (at leasts not by default).